FAQs.md

```diff
@@ -1,5 +1,9 @@
 ## Frequently Asked Questions
 
+**What is CORTX?** 
+
+Refer to the documentation [here](https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md) to know more about CORTX.
+
```

Comment:
please change this to say, 

CORTX is S3 compatible object storage developed by a community with a focused engineering effort provided by Seagate designed to allow the most economically efficient possible storage of massive data sets on mass capacity storage devices.  A brief introductory video is available [here](https://www.youtube.com/watch?v=dA-QtUBf16w&t=10s&ab_channel=SeagateTechnology) and please visit our entire [youtube channel](LINK) for more in-depth information.  At the heart of CORTX is the internal motr object system of which a detailed architecture description is available [here](https://github.com/Seagate/cortx-motr/blob/main/doc/motr-in-prose.md).  To ensure the best economics and the highest quality of code, CORTX is 100% open source with core components licensed as Apache and peripheral components licensed as LGPL.

Reply:
Done @johnbent, please approve.